test/utils/hooks.ts

```diff
@@ -44,26 +68,37 @@ export async function mochaGlobalSetup(this: GlobalFixturesContext) {
 }
 
 export async function mochaGlobalTeardown(this: GlobalFixturesContext) {
-  await this.compose.down();
+  await this.selenium.stop();
+  await this.node.stop();
 }
 
 export const mochaHooks = () => ({
   async beforeAll(this: mocha.Context) {
+    this.timeout(15 * 60 * 1000);
     this.wait = 15 * 1000;
     this.nodeUrl = 'http://waves-private-node:6869';
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    (global as any).browser = await remote({
+    const browser = await remote({
       logLevel: 'warn',
       capabilities: {
         browserName: 'chrome',
         'goog:chromeOptions': {
-          args: ['--load-extension=/app/dist/chrome', '--disable-web-security'],
+          args: [
+            '--load-extension=/app/dist/chrome',
+            '--disable-dev-shm-usage',
```

Comment:
`--disable-dev-shm-usage` option should be removed

Reply:
Done